Webapp failed to load

In chromium, I loaded mail.tutanota.com and just got a white page.

console error message: 
```
Uncaught (in promise) TypeError: Failed to fetch
Failed to load resource: net::ERR_FAILED
error Error: https://mail.tutanota.com/translation-translation-en-babf199f.js,  (SystemJS https://git.io/JvFET#3)
    at HTMLScriptElement.<anonymous> (polyfill.js:1)
common-min-82f59dc4.js:1 Unknown error Error: https://mail.tutanota.com/translation-translation-en-babf199f.js,  (SystemJS https://git.io/JvFET#3)
    at HTMLScriptElement.<anonymous> (polyfill.js:1)
2common-min-82f59dc4.js:1 error TypeError: Cannot read property 'unknownError_msg' of undefined
    at U.get (app.js:1)
    at U.getMaybeLazy (app.js:1)
    at gui-base-79a95423.js:1
    at H._execute (polyfill.js:1)
    at H._resolveFromExecutor (polyfill.js:1)
    at new H (polyfill.js:1)
    at Function.error (gui-base-79a95423.js:1)
    at main-3eb531cf.js:1
    at Object.$o [as handleUncaughtError] (main-3eb531cf.js:1)
    at app.js:1
    at l (polyfill.js:1)
    at H._settlePromiseFromHandler (polyfill.js:1)
    at H._settlePromise (polyfill.js:1)
    at H._settlePromise0 (polyfill.js:1)
    at H._settlePromises (polyfill.js:1)
    at f (polyfill.js:1)
```

It worked after a reload

--- a/dist.js
+++ b/dist.js
@@ -238,12 +238,7 @@ async function buildWebapp(version) {
 			return getChunkName(id, {getModuleInfo})
 		},
 		chunkFileNames: (chunkInfo) => {
-			// I would love to test chunkInfo.name but it will be just "en", not "translation-en" for some reason
-			if (chunkInfo.facadeModuleId && chunkInfo.facadeModuleId.includes("src/translations/")) {
-				return "translation-[name]-[hash].js"
-			} else {
 				return "[name]-[hash].js"
-			}
 		}
 	})
 	const chunks = output.output.map(c => c.fileName)

